### PR TITLE
CF-541 - Ensure Tiamat is kept in synch with standard-usage-schema

### PR DESCRIPTION
**Pull request by usnavi**
Adding tiamat jar as artifact published to maven

After this is accepted I'd like to cut a new release of nabu so I could verify my usage-schema work against the published jar.

### CODE REVIEW

File: tiamat/build.gradle [hunk 1]
```diff
@@ -147,6 +147,9 @@ artifacts {
         type   'rpm'
         builtBy buildRpm
     }
+    archives(jar) {
+        builtBy jar
+    }
```
Comment (shintasmith) [inline tiamat/build.gradle]: What does this do?

Comment (usnavi) [inline tiamat/build.gradle]: Adds the tiamat jar as one of the artifacts which is published to maven. 
I.e., https://docs.gradle.org/current/userguide/artifact_management.html

Comment (shintasmith) [inline tiamat/build.gradle]: Got it. Lgtm

**Comment by usnavi** [comment]
@shintasmith - noticed i could simplify the statement.

**Comment by shintasmith** [comment]
lgtm, can u ping me before u cut release?

**Pull request merged**